cfgov/unprocessed/css/organisms/email-signup.less

```diff
@@ -44,6 +44,15 @@
             .grid_column(12);
         }
     });
+
+    .m-form-field {
+        margin-bottom: unit( (@grid_gutter-width / 2) / @base-font-size-px, em);
+
+        .a-text-input {
+            margin-top: 0;
```

Comment:
Hmm, the margin on `m-form-field` is atomic, but this one isn't. Say you have molecule M that contains atom A.

```
---------
|   M   |
|       |
|  ---  |
| | A | |
|  ---  |
---------
```

If these are stacked they don't have margin:

 ```
---------
|   M   |
|       |
|  ---  |
| | A | |
|  ---  |
---------
---------
|   M   |
|       |
|  ---  |
| | A | |
|  ---  |
---------
```

So, putting that inside organism O and adjusting the margin is fine, because molecule M still equals molecule M's elsewhere in respect to its internal contents:

 ```
// Inside organism O
---------
|   M   |
|       |
|  ---  |
| | A | |
|  ---  |
---------

---------
|   M   |
|       |
|  ---  |
| | A | |
|  ---  |
---------
```

However, if inside O this happens:

```
// Inside organism O
---------
|   M   |
|  ---  |
| | A | |
|  ---  |
---------

---------
|   M   |
|  ---  |
| | A | |
|  ---  |
---------
```

Now molecule M does not equal molecule M elsewhere because it's internal contents have changed.

Do you see a way around this?

Reply:
If you can't find a way to make the margin unnecessary, we'd probably need a modifier class in CF for `m-form-field` that removed the margin.

Reply:
@anselmbradford It looks like the spacing between `.a-label__heading` and `.a-text-input` classes is a global issue -- their margins don't collapse, which results in 15px between heading-styled labels and their text inputs rather than the intended 10px. I've updated this spacing globally in https://github.com/cfpb/cfgov-refresh/pull/4005/commits/e0c19cce712833a1579582a95d44fe927078ef7d.